sql/updates/characters/2022_03_22_00_Update_Characters.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `bfa_characters`.`characters` 
+ADD COLUMN `camp_x` int(10) NULL DEFAULT NULL AFTER `numRespecs`,
+ADD COLUMN `camp_y` int(10) NULL DEFAULT NULL AFTER `camp_x`,
+ADD COLUMN `camp_z` int(10) NULL DEFAULT NULL AFTER `camp_y`,
+ADD COLUMN `camp_o` int(10) NULL DEFAULT NULL AFTER `camp_z`,
+ADD COLUMN `camp_mapid` int(10) NULL DEFAULT NULL AFTER `camp_o`;
```

Comment:
Just saw that i actually build in a database specification by myself ^^ will fix that later

Reply:
A column of coordinates must always be float, this way it would be fine:
ALTER TABLE `characters` 
ADD COLUMN `camp_x` float NULL DEFAULT NULL AFTER `numRespecs`,
ADD COLUMN `camp_y` float NULL DEFAULT NULL AFTER `camp_x`,
ADD COLUMN `camp_z` float NULL DEFAULT NULL AFTER `camp_y`,
ADD COLUMN `camp_o` float NULL DEFAULT NULL AFTER `camp_z`,
ADD COLUMN `camp_mapid` smallint(5) NULL DEFAULT NULL AFTER `camp_o`;

Reply:
makes totaly sence yes

Reply:
But mapid is not a float but an int so it is declared as smallint
ADD COLUMN `camp_mapid` float NULL DEFAULT NULL AFTER `camp_o`;